packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+
+Assign a custom Role by creating a Binding between the Role and the admin targetting an existing Resource Set.
+
+#### Request parameters
+
+| Parameter      | Description                                                   | Param Type    | DataType       | Required |
+| :------------- | :------------------------------------------------------------ | :------------ | :------------- | :------- |
+| role           | `id` of the Role                                              | Body          | String         | TRUE     |
+| members        | hrefs pointing to User(s) and/or Group(s) receiving the Role  | Body          | Array of hrefs | TRUE     |
+
+#### Response parameters
+
+The following `_links` are returned:
+* `self` gets this Role's Binding within the Resource Set
+* `bindings` get a paginable list of Role Bindings in the Resource Set
+* `resource-set` gets Resource Set
+
+#### Request example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+      "role": "cr0Yq6IJxGIr0ouum0g3",
+      "members": [
+        "https://${yourOktaDomain}/api/v1/groups/00guaxWZ0AOa5NFAj0g3"
+      ]
+}' "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/bindings"
+```
+
+#### Response example
+
+```json
+{
+  "_links": {
+    "self": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/bindings/cr0Yq6IJxGIr0ouum0g3"
+    },
+    "bindings": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g/bindings"
+    },
+    "resource-set": {
+      "href": "https://${yourOktaDomain}/api/v1/iam/resource-sets/iamoJDFKaJxGIr0oamd9g"
+    }
+  }
+}
+```
+
+### Add more members to a binding
+<ApiLifecycle access="beta" />
+
+<ApiOperation method="patch" url="/api/v1/iam/resource-sets/${resourceSetId}/bindings/${roleId}/members" />
+
+Add more Members to a Role Binding already created in a Resource Set.
+
+#### Request parameters
+
+| Parameter      | Description                                                           | Param Type   | DataType       | Required |
+| :------------- | :-------------------------------------------------------------------- | :----------- | :------------- | :------- |
+| resourceSetId  | `id` of the target Resource Set                                       | URL          | String         | TRUE     |
+| roleId         | the `id` of the Role to grant                                         | URL          | String         | TRUE     |
+| additions      | Array of hrefs pointing to User(s) and/or Group(s) receiving the Role | Body         | Array of hrefs | TRUE     |
+
+#### Response parameters
+
+The following `_links` are returned:
+* `self` gets this Role's Binding within the Resource Set
+* `bindings` get a paginable list of Role Bindings in the Resource Set
+* `resource-set` gets Resource Set
```

Comment:
gets Resource Set > gets the Resource Set
Add a period at the end of each bullet item since each are sentences.